PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -341,6 +344,7 @@ def nanoAOD_customizeCommon(process):
         nanoAOD_addDeepDoubleX_switch = cms.untracked.bool(False),
         nanoAOD_addDeepDoubleXV2_switch = cms.untracked.bool(False),
         nanoAOD_addParticleNet_switch = cms.untracked.bool(False),
+        nanoAOD_addParticleNetMass_switch = cms.untracked.bool(True),
```

Comment:
so this nanoAOD_addParticleNetMass_switch always `True` here means that we will re-run always the regression and it's expensive (about 10% of the whole nanoAOD production).

line 347 should be set to False as default
then activate for the past (
i.e. nanoAOD_addParticleNetMass_switch = cms.untracked.bool(True), in  line 360 and 367)

and then finally add a block 
    run2_nanoAOD_106Xv2.toModify(
        nanoAOD_addDeepInfoAK8_switch,
        nanoAOD_addParticleNetMass_switch = True
        )

Reply:
@mariadalfonso I believe we need to re-run it until it becomes available in MiniAOD, so setting `False` as the default does not seem a future-proof way to go.

Reply:
the miniAOD as the come out from this master: will contain or not the particleNetMass ?

Reply:
Yes it will contain. I guess maybe I am a bit confused with the many modifiers in NanoAOD. Specifically, does the modifier `run2_nanoAOD_106Xv2` correspond specifically to NanoAOD V9, or generally any Nano (V10, V11...) as long as it runs over 106X MiniAODv2?

Reply:
Nano modifiers are used to identify the mini in input.

So for the UL we have in production:
MiniAODv1 --> these have been used as input for the nanov8 and so all recipes need to be called with run2_nanoAOD_106Xv1 --> when nano is being produced one call it v8
MiniAODv2 were started in December '20 -->nano recipes will need to be called with the run2_nanoAOD_106Xv2 -->  these nano will be also called v9

if NanoV10 will read MiniAODv2  or a newer mini we do not know.

Master anyway make fresh mini  with the particleNetMass included, so we can avoid to recalculate.

I hope it's clearer.

Said that, we will need to do a major cleanup soon to prepare Run3 config, so we can restore the False as default at that time. I leave to you.

Reply:
Thanks a lot for the clarification @mariadalfonso !

If you are fine then I would prefer to postpone the change to `False` for the future cleanup.

Reply:
ok, let's do like this